### PR TITLE
Feat: Import Fuel Code Data from TFRS to LCFS - 828

### LINKED ISSUE #828: LCFS - Import production Fuel code data into dev
**Describe the task**
Import Fuel code data from the TFRS production database into the LCFS dev database

**Purpose**
This will help provide more realistic testing. The data is not sensitive information.

**Acceptance Criteria**
- [ ] first
- [ ] second
- [ ] third

**Additional context**
- Add any other context about the task here.
- Or here

### PR DESCRIPTION
**Pull request by hamed-valiollahi**
This PR imports fuel code data from the TFRS production database into the LCFS dev database. It also includes a prototype for potential use in the data migration process.

Closes #828

### CODE REVIEW

File: etl/nifi_scripts/fuel_code.groovy [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            fuel_type_id                       : record.fuel_id,
+            feedstock                          : record.feedstock,
+            feedstock_location                 : record.feedstock_location,
+            feedstock_misc                     : record.feedstock_misc,
+            facility_nameplate_capacity        : record.facility_nameplate,
+            former_company                     : record.former_company,
+            notes                              : null,
+            create_date                        : record.create_timestamp,
+            update_date                        : record.update_timestamp,
+            create_user                        : null,
```
Comment (AlexZorkin) [inline etl/nifi_scripts/fuel_code.groovy:46]: We will have to figure out the proper order of operation on these table migrations so we can ensure that create_user, and update_user entries are transferred correctly. Something to think about later.

**Review by AlexZorkin — APPROVED** [review]
Interesting approach for our migration. This will be a useful tool to use locally with our prod db instances in order to ensure all the entities are moved across correctly. Let's have a demo of it working sometime this week. Nice work :)

File: etl/nifi_scripts/fuel_code.groovy (hunk 1 above)
Comment (hamed-valiollahi) [reply etl/nifi_scripts/fuel_code.groovy:46]: I agree with you. We’ll need to plan the migration more carefully.

**Comment by hamed-valiollahi** [comment]
> Interesting approach for our migration. This will be a useful tool to use locally with our prod db instances in order to ensure all the entities are moved across correctly. Let's have a demo of it working sometime this week. Nice work :)

Thanks, Alex, for your feedback! I’m happy the data migration solution worked. I’ll give the team a quick demo this week.

**Pull request merged**